libbeat/beat/beat.go

```diff
@@ -113,7 +113,7 @@ type Beat struct {
 type BeatConfig struct {
 	Output  map[string]*common.Config
 	Logging logp.Logging
-	Shipper publisher.ShipperConfig
+	Shipper publisher.ShipperConfig `config:",inline"`
```

Comment:
Another style one: I like to put inlined configs first/last in struct to have them 'stick' out a little.